magentoerpconnect/sale.py

```diff
@@ -482,6 +483,14 @@ def move(self, binding):
 
 
 @magento
+class SaleOrderBundleImport(ConnectorUnit):
```

Comment:
It is named as if it was an Importer but it is not. Could you rename it to BundleLineLinker?